src/Style/CheckboxStyle.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace PhpSchool\CliMenu\Style;
+
+class CheckboxStyle
+{
+    use ItemStyleTrait;
+
+    protected const DEFAULT_STYLES = [
+        'markerOn'      => '[✔] ',
+        'markerOff'     => '[ ] ',
+        'itemExtra'     => '✔',
+        'displaysExtra' => false,
+    ];
+
+    public function __construct()
+    {
+        $this->setMarkerOn(self::DEFAULT_STYLES['markerOn']);
```

Comment:
Doesn't this automatically set it as custom?

Reply:
Yes, but then it has `$this->custom = false;`